dashboard/config/videos.csv

```diff
@@ -1,293 +1,293 @@
-CSF_bb8_skinOverview_C-F,,,SJgje3OuTJY,https://s3.amazonaws.com/videos.code.org/csf/BB-8_2-5.mp4
-CSF_bb8_skinOverview_K-1,,,G8ZOXOOYcCQ,https://s3.amazonaws.com/videos.code.org/csf/BB-8_K-1.mp4
-CSF_bb8_loops_C-F,,,sDg32zIlHrM,https://s3.amazonaws.com/videos.code.org/csf/BB-8_repeat.mp4
-CSF_harvester_skinOverview_K-1,,,k74wMtOEUWo,https://s3.amazonaws.com/videos.code.org/csf/Harvester_K1.mp4
-CSF_harvester_loops_K1,,,reYUq-IhVD8,https://s3.amazonaws.com/videos.code.org/csf/HarvesterLoops_K1.mp4
-CSF_scrat_loops_K1,,,9xQmtTzuCEk,https://s3.amazonaws.com/videos.code.org/csf/ScratRepeat_K1.mp4
-CSF_scrat_loops_CF,,,ZGHtw3gjNB0,https://s3.amazonaws.com/videos.code.org/csf/Scrat_Repeat_2-5.mp4
-CSF_if_allskins,,,KpMTXwlU270,https://videos.code.org/levelbuilder/conditionals_1_sm-mp4.mp4
-CSF_ifelse_allskins,,,sO4UYCbTxxo,https://videos.code.org/levelbuilder/conditionals_12_sm-mp4.mp4
-CSF_ifelseifelse_allskins,,,ZgB0Wp-fShk,https://videos.code.org/levelbuilder/conditionals_123_sm-mp4.mp4
-CSF_loops_AB,,,DgL1-axtxmU,https://videos.code.org/levelbuilder/loops_ab_sm-mp4.mp4
-csd_gamelab_booleans,,,_j9nvYKaOVE,https://videos.code.org/levelbuilder/csd_booleans_sm-mp4.mp4
-csd_gamelab_conditional_statements,,,TuLIK7DcPWs,https://videos.code.org/levelbuilder/csd_conditionals_sm-mp4.mp4
-csd_gamelab_if_else,,,6m6FFUJsOzo,https://videos.code.org/levelbuilder/csd_ifelse_sm-mp4.mp4
-CSF_SpriteLabIntro_CDEF,,,l0B0gDuHHC8,https://videos.code.org/levelbuilder/introducingspritelab_sm-mp4.mp4
-CSF_Spritelab_MakeSprite_CDEF,,,nHq7_-Kpgc8,https://videos.code.org/levelbuilder/howtomakeasprite_sm-mp4.mp4
-CDEF_Video_SpritesInAction,,,vsjoWjyQLRU,https://videos.code.org/levelbuilder/spritesinaction_sm-mp4.mp4
-CSF_SpriteLab_SpritesInAction_CDEF,,,vsjoWjyQLRU,https://videos.code.org/levelbuilder/spritesinaction_sm-mp4.mp4
-CSF_StarWars_ProjectIntro,,,hS1xTfIbanA,https://videos.code.org/levelbuilder/events_r2d2_sm-mp4.mp4
-mc_2018_intro,,,92sMXSm4dIg,https://videos.code.org/levelbuilder/mc_2018_intro-mp4.mp4
-MC_2018_loops,,,_97FJL9Ljdc,https://videos.code.org/levelbuilder/mc_2018_loops-mp4.mp4
-mc_2018_conditionals,,,j8TWTqBNV7g,https://videos.code.org/levelbuilder/mc_2018_conditionals-mp4.mp4
-mc_2018_closing,,,JZjUGd8Dk_Y,https://videos.code.org/levelbuilder/mc_2018_closing-mp4.mp4
-hoc_dance_warmup,,,bVHSrWuROrk,https://videos.code.org/levelbuilder/danceparty_01warmup_sm-mp4.mp4
-hoc_dance_events,,,COxnCoiHafc,https://videos.code.org/levelbuilder/danceparty_02events_sm-mp4.mp4
-hoc_dance_properties,,,Dq3HovyOmvg,https://videos.code.org/levelbuilder/danceparty_03properties_sm-mp4.mp4
-hoc_dance_congrats,,,pIxdBhJb8lc,https://videos.code.org/levelbuilder/danceparty_04partyon_sm-mp4.mp4
-c_and_c_great_to_create,,,Ltiz1B_oa-E,https://videos.code.org/levelbuilder/c_and_c_great_to_create-mp4.mp4
-c_and_c_digital_sharing,,,6RFY5YnA8iY,https://videos.code.org/levelbuilder/c_and_c_digital_sharing-mp4.mp4
+Key,Name,Concepts,YoutubeCode,Download,Locale
```

Comment:
I scripted it but yes. Because our code ignores extra columns, it was easier and simpler to add it now and just set all the current videos to be in English so that my future changes can assume that this column exists (I won't merge that change until this one goes through a couple DTPs).

Reply:
Yeah that makes a ton of sense!